### PR TITLE
Added no search result image and message

### PR DESCRIPTION
**Pull request by akshay11298**
Fixes # (issue)
> Fixes issue #798 

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions, so we can reproduce. 
> Please also list any relevant details for your test configuration.

- Tested with single and multiple organizations, entered name which is neither name of any application or organization.

## Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/pages/Applications/index.tsx [hunk 1]
```diff
@@ -380,18 +387,31 @@ const ApplicationsSection = () => {
...
-      {userOrgs &&
-        userOrgs.map((organizationObject: any, index: number) => {
+  let organizationsListComponent;
+  if (userOrgs) {
+    if (
+      props.searchKeyword &&
+      props.searchKeyword.trim().length > 0 &&
+      userOrgs.length === 0
+    ) {
+      organizationsListComponent = (
```
Comment (satbir121) [inline app/client/src/pages/Applications/index.tsx:397]: There was no need to add an if else here. If `userOrgs.map` will not run if `userOrgs.length === 0`.

**Comment by satbir121** [comment]
The image needs to be shifted upwards by `150px`.

**Comment by satbir121** [comment]
Also, please pull the latest release.

**Comment by akshay11298** [comment]
Hi @satbir121, please refer to PR #1240 for this issue. This branch had many file changes so I have created a new branch and PR #1240. Thank you

**Pull request closed without merging**